project_event/models/project_task.py

```diff
@@ -8,6 +8,7 @@
 class Task(models.Model):
     _name = "project.task"
     _inherit = ['project.task']
+    _order = 'date_start asc'
```

Comment:
@jananjoy we dont need to order all the tasks and activities by date_start on all the tree views